scripts/ci/ci_build_dockerhub.sh

```diff
@@ -67,6 +67,25 @@ echo
 # shellcheck source=scripts/ci/_script_init.sh
 . "$( dirname "${BASH_SOURCE[0]}" )/_script_init.sh"
 
+DOCKER_IMAGE_SPEC="${DOCKERHUB_USER}/${DOCKERHUB_REPO}:${DOCKER_TAG}"
+
+IMAGE_CREATED_SINCE=$(docker image ls "${DOCKER_IMAGE_SPEC}" --format '{{ .CreatedSince }}')
+
+if [[ ${IMAGE_CREATED_SINCE} =~ .*days.* || ${IMAGE_CREATED_SINCE} =~ .*weeks.* ||
+      ${IMAGE_CREATED_SINCE} =~ .*months.* || ${IMAGE_CREATED_SINCE} =~ .*years.* ||
+      ${IMAGE_CREATED_SINCE} =~ [0-9][0-9]\ hours ]]; then
```

Comment:
Maybe we need the following:

```suggestion
      ${IMAGE_CREATED_SINCE} =~ [0-9]?\ hours ]]; then
```

Reply:
```bash
❯ export tta="9 hours"
❯ if [[ "$tta" =~ [0-9]?\ hours ]]; then
echo "Rebuilding the image."
else
echo "Skipping rebuilding the image."
fi
Rebuilding the image.
```